pyblish_qml/ipc/server.py

```diff
@@ -112,8 +112,9 @@ def show(self, settings=None):
 
     def hide(self):
         """Hide the GUI"""
-        self._dispatch("hide")
-        return self.vessel.hide()
+        success = self._dispatch("hide")
+        self.vessel.hide()
+        return success
 
```

Comment:
`eventFilter` will get removed if `proxy._dispatch` returns `False`/`None`, which suggest that subprocee connection has closed.
It's obvious that returning the result of `self.vessel.hide()` is incorrect.